lib/Onyx.d.ts

```diff
@@ -20,41 +19,6 @@ type NullableKeyValueMapping = {
     [TKey in OnyxKey]: OnyxEntry<KeyValueMapping[TKey]>;
 };
 
-/**
```

Comment:
I think you don't need to move these types to other file, just export from here (like it was done for `OnyxUpdate`).

Reply:
Thanks, done!